### ISSUE TITLE
gatsby-plugin-feed title not passed to rss utility

### ISSUE THREAD

**Issue by cueblermm** [gatsbyjs/gatsby#15595]
## Description

title option (as described here: [https://www.gatsbyjs.org/packages/gatsby-plugin-feed/](https://www.gatsbyjs.org/packages/gatsby-plugin-feed/)) is not passed to the rss utility

Possible fix:
```
- a/packages/gatsby-plugin-feed/src/internals.js
+++ b/packages/gatsby-plugin-feed/src/internals.js
@@ -29,8 +29,8 @@ export const defaultOptions = {
setup: ({
query: {
site: { siteMetadata },
- ...rest
},
+ ...rest
}) => {
return {
...siteMetadata,
```

### Steps to reproduce
gatsby-config.js:
```
{
      resolve: 'gatsby-plugin-feed',
      options: {
        query: `
          {
            site {
              siteMetadata {
                siteTitle
                siteUrl
                site_url: siteUrl
              }
            }
          }
        `,
        feeds: [
          {
            serialize: ({
              query: {
                site, allMarkdownRemark
              }
            }) => {
              return allMarkdownRemark.edges.map(edge => {
                return Object.assign({}, edge.node.frontmatter, {
                  description: edge.node.excerpt,
                  date: edge.node.frontmatter.date,
                  url: site.siteMetadata.siteUrl + edge.node.fields.slug,
                  guid: site.siteMetadata.siteUrl + edge.node.fields.slug,
                  custom_elements: [{
                    'content:encoded': edge.node.html
                  }]
                });
              });
            },
            query: `
              {
                allMarkdownRemark(
                  sort: { order: DESC, fields: [frontmatter___date] },
                ) {
                  edges {
                    node {
                      excerpt
                      html
                      fields { slug }
                      frontmatter {
                        title
                        date
                      }
                    }
                  }
                }
              }
            `,
            // title not passed to rss
            title: 'My Title',
            output: '/feed.xml'
          }
        ]
      }
}

````

### Expected result

"My Title" is the title of the rss feed

### Actual result

"Untitled RSS Feed" is the title of the rss feed

### Environment

  System:
    OS: Windows 7
    CPU: (4) x64 Intel(R) Core(TM) i5 CPU       M 520  @ 2.40GHz
  Binaries:
    Yarn: 1.16.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 3.7.1 - C:\Users\User\AppData\Local\Programs\Python\Python37\python.EXE
  npmPackages:
    gatsby: 2.13.12 => 2.13.12 
    gatsby-cli: 2.7.8 => 2.7.8 
    gatsby-image: 2.2.4 => 2.2.4 
    gatsby-paginate: 1.1.0 => 1.1.0 
    gatsby-plugin-canonical-urls: 2.1.0 => 2.1.0 
    gatsby-plugin-eslint: 2.0.5 => 2.0.5 
    gatsby-plugin-feed: 2.3.2 => 2.3.2 
    gatsby-plugin-google-analytics: 2.1.1 => 2.1.1 
    gatsby-plugin-html-attributes: 1.0.5 => 1.0.5 
    gatsby-plugin-manifest: 2.2.1 => 2.2.1 
    gatsby-plugin-meta-redirect: 1.1.1 => 1.1.1 
    gatsby-plugin-netlify: 2.1.0 => 2.1.0 
    gatsby-plugin-netlify-cms: 4.1.1 => 4.1.1 
    gatsby-plugin-offline: 2.2.1 => 2.2.1 
    gatsby-plugin-react-helmet: 3.1.0 => 3.1.0 
    gatsby-plugin-react-svg: 2.1.1 => 2.1.1 
    gatsby-plugin-sass: 2.1.0 => 2.1.0 
    gatsby-plugin-sharp: 2.2.3 => 2.2.3 
    gatsby-plugin-sitemap: 2.2.1 => 2.2.1 
    gatsby-plugin-styled-components: 3.1.0 => 3.1.0 
    gatsby-plugin-webpack-size: 1.0.0 => 1.0.0 
    gatsby-redirect-from: 0.2.1 => 0.2.1 
    gatsby-remark-embed-video: 1.7.1 => 1.7.1 
    gatsby-remark-images: 3.1.3 => 3.1.3 
    gatsby-remark-relative-images: 0.2.2 => 0.2.2 
    gatsby-source-filesystem: 2.1.3 => 2.1.3 
    gatsby-transformer-json: 2.2.0 => 2.2.0 
    gatsby-transformer-remark: 2.6.1 => 2.6.1 
    gatsby-transformer-sharp: 2.2.1 => 2.2.1

**Comment by sergkudinov** [comment]
The same question. And in additional I am trying to set <language> tag, but it doesn't put this into the rss.xml file.

It is written here that all additional options are passed through to the [rss utility](https://www.npmjs.com/package/rss#itemoptions), but it doesn't work for me. How to use it?

**Comment by GerardKetuma** [comment]
Having the same issue. RSS Feed title is not being passed to the output. Getting 'Untitled RSS Feed'

**Comment by sidharthachatterjee** [comment]
This was fixed in https://github.com/gatsbyjs/gatsby/pull/16814 and published in `gatsby-plugin-feed@2.3.8`
